osbs/api.py

```diff
@@ -553,6 +558,29 @@ def create_worker_build(self, *args, **kwargs):
         return self.create_prod_build(*args, **kwargs)
 
     @osbsapi
+    def create_orchestrator_build(self, *args, **kwargs):
+        """
+        Create an orchestrator build
+
+        Pass through method to create_prod_build with the following
+        modifications:
+            - platforms param is required
+            - inner template set to orchestrator_inner.json if not set
+            - outer template set to orchestrator.json if not set
+            - customize configuration set to orchestrator_customize.json if not set
+
+        :return: BuildResponse instance
+        """
+        if not kwargs.get('platforms'):
```

Comment:
I don't think it should be. The `orchestrate_build` plugin will essentially be a no op in such case. I don't really see a valid uses case for `[]`.